.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This is VSCode specific file, you can add `.vscode` directory to the .gitignore, so we won't track it anymore.

Reply:
Done. But don't know how it appeared here, i didn't add it to commit.

Reply:
Maybe when adding all files in the directory, example by `git add .` which adds all file including untracked files into commit staging. It's easy to miss that, in my case a lot of files related to my system used to be added and some of them already supposedly should be listed in `.gitignore`.

Reply:
> 
> 
> This comment is still valid.

When I was committing git status didn't show anything this time. Anyway, I corrected my mistake and added .vscode folder to .gitignore instead of settings file. Hope it will resolve this.

Reply:
Yes, it didn't because you already had the file in previous commits, when adding file and/or directory to the `.gitignore` it doesn't automatically hint you that you might want to remove the file. It basically prevent future files in the directory to be accidentally commited. So you basically want to manually delete the file, after updating `.gitignore` to ensure the old version and/or furhter modification, won't be pushed in future.

Reply:
> So you basically want to manually delete the file, after updating `.gitignore` to ensure the old version and/or furhter modification, won't be pushed in future.

In settings.json I thought there is useful property of `"editor.tabSize": 2`. Anyway I removed it and it has no effect on tabbing. It is still 2 spaces.

Reply:
I see your point, it is a useful property for your use-case, but there are multiple reasons why it's not a good idea to commit your local IDE settings:

1. As soon me/you/other contributor add any new setting in the local `.vscode` setting, they will still appear in the staging directory, so the person will end up adding new settings to the commit accidentally.
2. Those are VSCode related settings, but it doesn't necessarily mean that other contributors might use VSCode as well and that might eventually lead into maintaining multiple IDE specific settings.

I think what might have been not clear here is that after removing this file from the commit, you can still have your local settings in `.vscode` folder and/or have a global settings, unless you delete the folder yourself. So basically, after removing this file from the commit, please set that setting in your IDE again and they should stay there.

I would rather rely on the linters to pick up similar issue, rather than enforcing IDE specific rules. The problem is with [our linter](https://github.com/browser-automation/cba-website/issues/16), but not with the IDE.

It's a great idea of yours to have best practices for VSCode and/or other IDEs, as an idea we can have a [WIKI page for that with the recommended settings and/or general best practises](https://github.com/browser-automation/cba-website/wiki), or a new repository with the various VSCode settings, that are common for [browser-automation organization](https://github.com/browser-automation), but as we haven't identified a common repository best practices yet, wiki might be a good start.